ServerCore/FileManager.cs

```diff
@@ -103,16 +107,24 @@ public static async Task DeleteBlobAsync(Uri fileUri)
         /// Gets a reference to a random directory
         /// </summary>
         /// <param name="eventId">Event the directory should be associated with</param>
-        private static async Task<CloudBlobDirectory> GetRandomDirectoryAsync(int eventId)
+        private static async Task<CloudBlobDirectory> GetRandomDirectoryAsync(int eventId, bool randomize)
```

Comment:
I think it would be better to not pass a flag here, and instead move the shortcut behavior directly into UploadBlobsAsync. Otherwise, "randomize without randomize" is hard to process.

Reply:
Good point, fixed.